File: rafael/funcoes-ou-metodos/AnoBissexto.java
class AnoBissexto {
	public static void main(String[] args) {
		boolean b = bissexto(2000);
		System.out.println("2000 " + b);

		b = bissexto(2012);
		System.out.println("2012 " + b);

		b = bissexto(2025);
		System.out.println("2025 " + b);

		b = bissexto(2100);
		System.out.println("2100 " + b);
	}

	static boolean bissexto(int ano){
		return ano % 400 == 0 || (ano % 100 != 0 && ano % 4 == 0);
	}
}

File: rafael/funcoes-ou-metodos/AnoBissexto.cs
class AnoBissexto 
{
	static void Main() 
	{
		bool b = bissexto(2000);
		System.Console.WriteLine("2000 " + b);

		b = bissexto(2012);
		System.Console.WriteLine("2012 " + b);

		b = bissexto(2025);
		System.Console.WriteLine("2025 " + b);

		b = bissexto(2100);
		System.Console.WriteLine("2100 " + b);
	}

	static bool bissexto(int ano)
	{
		return ano % 400 == 0 || (ano % 100 != 0 && ano % 4 == 0);
	}
}
